packages/test/local-server-stress-tests/src/test/localServerStressOrderSequentially.spec.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { done, isOperationType, takeAsync } from "@fluid-private/stochastic-test-utils";
+
+import { makeGenerator, reducer, saveFailures, type StressOperations } from "../baseModel.js";
+import {
+	convertToRealHandles,
+	covertLocalServerStateToDdsState,
+	DDSModelOpGenerator,
+	loadAllHandles,
+	validateConsistencyOfAllDDS,
+	type DDSModelOp,
+	type OrderSequentially,
+} from "../ddsOperations.js";
+import {
+	createLocalServerStressSuite,
+	LocalServerStressModel,
+	type LocalServerStressState,
+} from "../localServerStressHarness.js";
+
+const orderSequentiallyReducer = async (
+	state: LocalServerStressState,
+	op: OrderSequentially,
+) => {
+	const { baseModel, taggedHandles } = await loadAllHandles(state);
+	const ddsState = await covertLocalServerStateToDdsState(state);
+	const rollbackError = new Error("rollback");
+	try {
+		state.datastore.orderSequentially(() => {
+			for (const o of op.operations) {
+				baseModel.reducer(ddsState, convertToRealHandles(o, taggedHandles));
+				if (op.rollback) {
+					throw rollbackError;
```

Comment:
So if there are 3 operations on the OrderSequentillay operation, we'll rollback after each one?  I would think maybe we'd rather choose to rollback once at the end?

Reply:
Ok I see the comment later that says we can only put one op in here.  Should we just update the type for `OrderSequentially` to just have one operation then, rather than an array?

Reply:
this is a typo. fixed it